node/src/command.rs

```diff
@@ -12,7 +12,7 @@ use sp_keyring::Sr25519Keyring;
 
 impl SubstrateCli for Cli {
     fn impl_name() -> String {
-        "DSCP Node".into()
+        "DSCP Node ".into()
```

Comment:
why the extra space?

Reply:
`Error:   --> /home/runner/work/dscp-node/dscp-node/node/src/command.rs:15:15
   |
15 |         'DSCP Node'.into()
   |               ^^^^ unknown prefix
   |
   = note: prefixed identifiers and literals are reserved since Rust 2021
help: consider inserting whitespace here
   |
15 |         'DSCP Node '.into()
   |                   +
`

Reply:
that's because of single-quotes not because of the space. Line should be
```rust
"DSCP Node".into()
```
which builds fine for me locally